x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -209,7 +214,12 @@ export const uiQueryParams: (
 
         if (value !== undefined) {
           if (key === 'show') {
-            if (value === 'policy_response' || value === 'details' || value === 'isolate') {
+            if (
```

Comment:
You can disregard @pzl 's request here. I already did this in my PR for un-isolate. Once you merge this in I will add the additional show value to my PR